jetty-util/src/main/java/org/eclipse/jetty/util/Pool.java

```diff
@@ -604,8 +604,8 @@ boolean tryRelease()
         }
 
         /**
-         * Try to mark the entry as removed.
-         * @return true if the entry has to be removed from the containing pool, false otherwise.
+         * Mark the entry as closed.
```

Comment:
This change is confusing as says that it "mark" but actually "tries to mark", says "closed", but the method is called "tryRemove".
I would revert this change and perhaps explain a little more in the javadocs what the method actually does.

Reply:
Damn, I replaced incomplete doc with with even worse doc! Let me rework this.